Quest db (#1222)
n8n-io/n8n-docs: docs/integrations/builtin/app-nodes/n8n-nodes-base.questdb.md

# QuestDB

> The QuestDB node allows you to automate work in QuestDB, and integrate QuestDB with other applications. n8n has built-in support for a wide range of QuestDB features, including executing a SQL query and inserting rows in database. 

Reviewer: ```suggestion
The QuestDB node allows you to automate work in QuestDB, and integrate QuestDB with other applications. n8n supports executing an SQL query and inserting rows in a database with QuestDB.
```

On this page, you'll find a list of operations the QuestDB node supports and links to more resources.

!!! note "Credentials"

>     Refer to [QuestDB credentials](https://docs.n8n.io/integrations/builtin/credentials/questdb/) for guidance on setting up authentication. 

Reviewer: ```suggestion
    Refer to [QuestDB credentials](/integrations/builtin/credentials/questdb/) for guidance on setting up authentication. 
```
